src/GraphQLParser.Tests/ParserTests.cs

```diff
@@ -47,8 +47,9 @@ public void Extra_Comments_Should_Read_Correctly(IgnoreOptions options)
 
         [Theory]
         //[InlineData(IgnoreOptions.None)]
-        [InlineData(IgnoreOptions.IgnoreComments)]
-        [InlineData(IgnoreOptions.IgnoreCommentsAndLocations)]
+        [InlineData(IgnoreOptions.Comments)]
```

Comment:
Changes should be made on various tests (e.g. the `Comments_on_FragmentSpread_Should_Read_Correclty` test) to include `[InlineData(IgnoreOptions.Locations)]` to test reading comments when only locations are ignored

Reply:
(Not this test, but I couldn't comment on line 69)

Reply:
Yes